StripesFormWrapper.js

```diff
@@ -51,7 +51,10 @@ class StripesFormWrapper extends Component {
 
   continue() {
     this.unblock();
-    this.props.history.push(this.state.nextLocation.pathname);
+    this.props.history.goBack();
+    setTimeout(() => {
+      this.props.history.push(this.state.nextLocation.pathname);
+    }, 1);
```

Comment:
Why call `setTimeout` here?

Reply:
Right. We're doing a very bad thing here. We're going back to the previous page, hoping that stripes-core gets a chance to fire and register this as the most recent URL visited within the present app, and _then_ shooting off to another app. I don't see how anythssibly go wrong.ing could po

Reply:
Yeah but it works :)

Reply:
@MikeTaylor this is a temporary hack/fix. I think we will need invest more time to fix it properly.  Perhaps after q3.

Reply:
Ideally if we could access `lastVisited` for given module right from `stripes-form` then we could control have more control here.

Reply:
Here's how we handled this in `ui-eholdings`; we don't use `stripes-form` at all: https://github.com/folio-org/ui-eholdings/blob/eed25373fd3fa173922409fba6f7b21de5eea5b5/src/components/navigation-modal/navigation-modal.js#L52-L72

Reply:
@cherewaty not related to this but I just tried to reproduce it from eholdings on folio-snapshot and when I try to edit one of the items I get: 

"ERROR:Cannot read property 'attributes' of undefined"